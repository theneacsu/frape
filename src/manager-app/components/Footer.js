import React from 'react'

const Footer = () => (
  <div className="footer">
    Made with{' '}
    <img className="footer-icon" src="images/like.png" alt="heart icon" /> and{' '}
    <img className="footer-icon" src="images/coffee.png" alt="coffee icon" /> by
    Neacsu Alexandru
  </div>
)

export default Footer
