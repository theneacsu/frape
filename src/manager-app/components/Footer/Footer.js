import React from 'react'
import styles from './Footer.module.css'

const Footer = () => (
  <div className={styles.div}>
    Made with{' '}
    <img className={styles.icon} src="images/like.png" alt="heart icon" /> and{' '}
    <img className={styles.icon} src="images/coffee.png" alt="coffee icon" /> by
    Neacsu Alexandru
  </div>
)

export default Footer
