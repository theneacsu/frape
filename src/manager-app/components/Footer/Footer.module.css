.div {
  text-align: center;
  font-size: 1.7rem;
  font-style: italic;
  font-weight: 100;
  padding: 1rem;
  background-color: #4cb69f;
  color: white;
}

.icon {
  height: 1.7rem;
}
