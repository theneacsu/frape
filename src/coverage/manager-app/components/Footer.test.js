import React from 'react'
import { shallow } from 'enzyme'
import Footer from '../../../manager-app/components/Footer'

test('Should render correctly the footer component', () => {
  const wrapper = shallow(<Footer />)
  expect(wrapper).toMatchSnapshot()
})
