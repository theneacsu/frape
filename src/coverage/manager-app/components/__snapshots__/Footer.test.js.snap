// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly the footer component 1`] = `
<div
  className="footer"
>
  Made with
   
  <img
    alt="heart icon"
    className="footer-icon"
    src="images/like.png"
  />
   and
   
  <img
    alt="coffee icon"
    className="footer-icon"
    src="images/coffee.png"
  />
   by Neacsu Alexandru
</div>
`;
